test/e2e/storage/vsphere/vsphere_utils.go

```diff
@@ -467,3 +474,97 @@ func getVSphere(c clientset.Interface) (*vsphere.VSphere, error) {
 func GetVSphere(c clientset.Interface) (*vsphere.VSphere, error) {
 	return getVSphere(c)
 }
+
+// get .vmx file path for a virtual machine
+func getVMXFilePath(vmObject *object.VirtualMachine) (vmxPath string) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	var nodeVM mo.VirtualMachine
+	err := vmObject.Properties(ctx, vmObject.Reference(), []string{"config.files"}, &nodeVM)
+	Expect(err).NotTo(HaveOccurred())
+	Expect(nodeVM.Config).NotTo(BeNil())
+
+	vmxPath = nodeVM.Config.Files.VmPathName
+	framework.Logf("vmx file path is %s", vmxPath)
+	return vmxPath
+}
+
+// verify ready node count. Try upto 3 minutes. If count is expected count, return the nodeList
+func verifyReadyNodeCount(client clientset.Interface, expectedNodes int) (nodeList *v1.NodeList) {
```

Comment:
Method name starting with 'verify' gives an impression that return type would be bool or some error will be thrown in case verification goes wrong. If you have to return NodeList from this method, please rename to a appropriately.

Reply:
Limiting this util to only verify the expected ready node count and return bool.